src/SingleCellAnnotator.imjoy.html

```diff
@@ -468,9 +496,16 @@
       const allFeatures = this.vector_source.getFeatures();
       const format = new ol.format.GeoJSON();
       const routeFeatures = format.writeFeaturesObject(allFeatures, {decimals: 1});
+      if (!routeFeatures.features.length){
```

Comment:
We can perhaps disable the save button, only enable it when the user redraw the annotation.

Reply:
done